src/util/github/PullRequestHelper.ts

```diff
@@ -0,0 +1,17 @@
+import GitHubHelper from "./GitHubHelper";
+
+export default class PullRequestHelper extends GitHubHelper {
+  public getPRDescripition() {
+    return this.context.payload.pull_request.body
```

Comment:
This is `null` when the PR description is empty, if we return an empty string when `null` then we could fix #21 😁

Reply:
Sweet, that his fixed it! 😁